core/domain/exp_domain.py

```diff
@@ -3179,6 +3335,57 @@ def get_all_html_content_strings(self):
 
         return html_list
 
+    @classmethod
+    def get_image_counter(cls, states_dict):
+        """Gets an image counter of the exploration.
+
+        Returns:
+            image_counter: int. The image counter.
+        """
+        # exploration_html = (self.get_all_html_content_strings())[0]
+        exploration_html = ''
```

Comment:
Instead of storing the HTML data try storing the image count.

Reply:
done